api/test/wfl/integration/modules/covid_test.clj

```diff
@@ -2,18 +2,61 @@
   (:require [clojure.test :refer :all]
             [clojure.test :as clj-test]
             [wfl.tools.fixtures :as fixtures]
-            [wfl.jdbc :as jdbc]
-            [wfl.api.workloads :as workloads]))
+            [wfl.tools.workloads :as workloads]))
 
 (clj-test/use-fixtures :once fixtures/temporary-postgresql-database)
 
-(deftest test-create-covid-workflow
-  (jdbc/with-db-transaction [tx (fixtures/testing-db-config)]
-                            (workloads/create-workload! tx {:pipeline "Sarscov2IlluminaFull"
-                                                            :source {:name "Terra DataRepo"}})
-                            )
-  )
+(def ^:private testing-dataset "ff6e2b40-6497-4340-8947-2f52a658f561")
+(def ^:private testing-workspace "general-dev-billing-account/test-snapshots")
+(def ^:private testing-method-configuration "pathogen-genomic-surveillance/sarscov2_illumina_full")
+
+(defn ^:private make-covid-workload-request []
+  (-> (workloads/covid-workload-request testing-dataset
+                                        testing-method-configuration
+                                        testing-workspace)
+      (assoc :creator @workloads/email)))
+
+(deftest test-create-covid-workload
+  (testing "That COVID workload-request can pass verification"
+    (let [workload (workloads/create-workload! (make-covid-workload-request))]
+      ;; NOTE: COVID workload creation returns nil until it's fully implemented
+      (is (nil? workload))))
```

Comment:
@okotsopoulos I think https://github.com/broadinstitute/wfl/pull/379/files#r625164593 is to make sure the verify source does not return anything. Basically it wants the side effect but not the result. (not sure if we could use `run!` to simplify)

Reply:
Ahh, okay.  So the anonymous function change in the code was to make integration testing easier?

Reply:
Yeah, but after looking it twice, I think we don't really care about the nullity of the returned result… in the test we should just say `(is workload)`. @rhiananthony

Reply:
We removed it as Tom showed us the way #mandelorian